examples/server-example.py

```diff
@@ -74,11 +72,26 @@ def multiply(parent, x, y):
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # get Objects node, this is where we should put our custom stuff
+
+    # create a new node type we can instantiate in our address space
+    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
```

Comment:
Either is fine for me. I was just thinking that it might be cleaner to get a few "constants" when the server is created. If you just got a reference to the base type node on init, we do not need more methods in the server class.

However I don't know if that is a proper python convention.